src/highlighters.cc

```diff
@@ -1141,8 +1150,12 @@ struct LineNumbersHighlighter : Highlighter
             snprintf(buffer, 16, format, std::abs(line_to_format));
             const auto atom_face = last_line == current_line ? face_wrapped :
                 ((m_hl_cursor_line and is_cursor_line) ? face_absolute : face);
+
+            const auto& separator {is_cursor_line && m_separator_cursor && last_line != current_line
```

Comment:
its a bit weird to use a brace base init here, at least its not very consistent with the codebase, we typically use `=` to assign references.